modules/src/lib.rs

```diff
@@ -6,7 +6,7 @@
     trivial_casts,
     trivial_numeric_casts,
     unused_import_braces,
-    unused_qualifications,
+    // unused_qualifications,
```

Comment:
What's the reason for commenting this out?

Reply:
It has to do with the generated Rust code which `prost` generated, for instance: `pub versions: ::std::vec::Vec<std::string::String>`. This seems to be the offending part that triggers the unused_qualifications warning, but I have not digged into it.

Any thoughts or prior experience with unused_qualifications ? I almost forgot about this. We should probably fix & enable the warning again it before merge.